compiler/rustc_target/src/spec/mod.rs

```diff
@@ -502,6 +502,26 @@ impl SplitDebuginfo {
             SplitDebuginfo::Unpacked => "unpacked",
         }
     }
+
+    /// Returns `true` if the split debuginfo configuration indicated by `self` is stable on
+    /// `target`.
+    pub fn is_stable_on_platform(&self, target: &TargetOptions) -> bool {
+        // Off, unpacked and packed split debuginfo are stable on OS X-like platforms.
+        if target.is_like_osx {
+            return true;
+        }
+
+        // Packed split debuginfo is the only valid option on Windows platforms.
+        if target.is_like_windows {
+            return match self {
+                SplitDebuginfo::Packed => true,
+                SplitDebuginfo::Off | SplitDebuginfo::Unpacked => false,
+            };
+        }
```

Comment:
I'm not sure this is correct for the `*-windows-gnu` targets. IIRC they use DWARF which, I assume, means they could take full advantage of the Split DWARF feature.

Reply:
Discussed this [in Zulip](https://rust-lang.zulipchat.com/#narrow/stream/317568-t-compiler.2Fwg-debugging/topic/DWARF.20or.20PDB.20on.20Windows/near/285946786). We're going to stabilize `-Csplit-debuginfo=packed` for Windows MSVC targets and `-Csplit-debuginfo=off` for Windows GNU targets. Other values for the option remain unstable on Windows. We'll probably need to do some work to support Split DWARF on Windows GNU but that's left for a future task.